uwblueprint/focus-on-nature
# feat: add waiver sections

## Notion ticket link

[Form Management: Waiver - Add Waiver Sections](https://www.notion.so/uwblueprintexecs/Form-Management-Waiver-Add-Waiver-Sections-1e8089e7c66845219def2fa836f6b388)


## Implementation description
* moved `GlobalFormsPage` to `index.ts` in GlobalForms folder
* created `Footer` component in GlobalForms folder 
* created `WaiverAddSectionModal` to add waiver clauses 
* created `AdminAPIClient` to send waiver update requests

https://user-images.githubusercontent.com/46568041/196001554-2bdfe6f6-4beb-4b27-b861-c77ffb30f859.mov


## Steps to test
1. go to global forms page 
2. click on "add form section" button in the bottom right corner 
3. add a new clause 
4. verify the toast message shows up 
5. check MongoDB to see that the waiver has been updated 


## What should reviewers focus on?
* is calling `onAddWaiverSectionClick` in `index.ts` the best place to do it? 
* I put it there because I need access to the initial `waiver` data to append clauses to it
* I didn't want to call the function in the Footer because that felt weird + I'd need to pass in the waiver data as a prop only to append clauses in the function call 
* I also didn't want to call the function in a new "WaiverPage" component because I'd somehow have to make the modal callback in a sibling component 


## Checklist
